extensions/wikia/PhalanxII/models/PhalanxUserModel.php

```diff
@@ -42,12 +42,44 @@ public function getText() {
 		return $ret;
 	}
 
+    /**
+     * Format the message with user block reason. If additional description was provided when creating that block,
+     * it's included in the message.
+     *
+     * @param String $reason Optional reason that came with the block
+     * @param bool $isExact Set to true for exact block
+     * @param bool $isBlockIP Set to true for IP blocks
+     * @return String Translated message with optional reason details
+     */
+    protected function getBlockReasonMessage( $reason, $isExact, $isBlockIP ) {
+        if ( $reason ) {
+            // a reason was given
+            if ( $isExact ) {
+                $result = wfMsg( 'phalanx-user-block-withreason-exact', $reason );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this legacy wfMsg* function call with one of the new i18n functions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3ALegacyWfMsgUsage)